cmd/user-list/main.go

```diff
@@ -17,8 +19,8 @@ var args struct {
 	config.ConfigArgs
 
 	Emails []string `help:"Optional list of users to limit the results to (leave empty for all users)." arg:"" optional:""`
```

Comment:
I don't like this option.
It looks like you are relying on old (pre user split) code for an example, but that will probably lead you down a bad road.

Instead, you should be looking towards the Python interface's behavior as an example:
https://github.com/edulinq/autograder-py/blob/main/autograder/cli/users/list.py
(Since this was re-written for the new user model.)

But you can keep it even simpler, since only admins will be using this and we trust them to have basic knowledge.
So I would leave out niceties like the `--normalize` flag (just normalize by default).